tasks/HandMeThat.tex

```diff
@@ -61,7 +61,7 @@ \subsection*{Additional rules and remarks}
 
 	\item \textbf{Incorrect guesses:} Incorrect guesses reduce the value of the correct guess by 200 points, each, the first two times. Guessing correctly on the third or fourth attempt is worth 100 points. After the fourth guess is worth no points.
 
-	\item\textbf{Colors and categories:} Asking for the color or category of a pointed object applies a penalty of 400 points for that particular object.
```

Comment:
I would not remove this rule, otherwise the test becomes trivial

Reply:
This is converted to the clarifying questions. So asking for color is also a reduction of 150.